examples/decoding/plot_receptive_field.py

```diff
@@ -5,9 +5,11 @@
 
 This example reproduces figures from Lalor et al's mTRF toolbox in
 matlab [1]_. We will show how the :class:`mne.decoding.ReceptiveField` class
-can perform a similar function along with scikit-learn. We will fit a
-linear encoding model using the continuously-varying speech envelope to
-predict activity of a 128 channel EEG system.
+can perform a similar function along with scikit-learn. We will first fit a
+linear encoding (or forward) model using the continuously-varying speech
```

Comment:
You are going to get an angry GIF from @choldgraf, I had a similar "encoding (or forward)" in my PR and it did not go over well

Reply:
Actually I am fine with it as long as they're both specified, and as long as `encoding/decoding` come first ;-)

Reply:
also now I feel like a curmudgeonly old academic